generator/honda/honda_civic_ex_2022_can.dbc

```diff
@@ -50,6 +50,11 @@ BO_ 467 CRUISE_FAULT_STATUS: 8 XXX
  SG_ CHECKSUM : 59|4@0+ (1,0) [0|15] "" XXX
  SG_ COUNTER : 61|2@0+ (1,0) [0|3] "" XXX
 
+BO_ 477 GEARBOX_BOH: 8 XXX
```

Comment:
Can you give this a better name?

Reply:
GEARBOX_ALT_2, perhaps?  Or could do GEARBOX_ALT_A and GEARBOX_ALT_B?

Reply:
Depends on where this comes from, Toyota calls a similar message `CLUTCH`. This can be `GEARBOX_MANUAL`

Reply:
I don't want to do gearbox_manual or gearbox_mt as this message is also found on the CVT transmission vehicles. The GEAR_MT signal only seems to change on MT vehicles while it's always 0 on CVT transmission vehicles. I will go with ALT_2 in order to prevent additional changes needed to support the renaming

Reply:
Updated in [c1e1823](https://github.com/commaai/opendbc/pull/1034/commits/c1e1823d7de3662a629c0251d90f6daac69c4f08)

Reply:
if it exists but is truly unused, those names seem fine to me. it's not clear what this is for from reading the DBC

Reply:
As I recall, on the CVT vehicles, the message itself has data in other bytes, however only on the MT vehicles  does GEAR_MT actually have data. On CVT vehicles GEAR_MT is always zero. I'm working on getting access again to the CVT integra device that I can confirm this. 

You are saying though that even if there is data in other bytes on the CVT vehicles that you are fine naming the message `GEARBOX_MT` or `GEARBOX_MANUAL` ?

Reply:
From this route on a CVT integra you can see that there is other data on this message, but the `GEAR_MT` signal is always 0. `3f8ae015ce70365f/000000fe--b691f954c2`

<img width="1111" alt="image" src="https://github.com/user-attachments/assets/70902d73-4216-410c-b6a0-076c8fc0edc9">

Reply:
@sshane any more comments on this?